internal/hook/item_hook_handler_test.go

```diff
@@ -490,11 +490,10 @@ func TestHandleHooks(t *testing.T) {
 			err := h.HandleHooks(velerotest.NewLogger(), groupResource, test.item, test.hooks, test.phase, hookTracker)
 
 			if test.expectedError != nil {
-				assert.EqualError(t, err, test.expectedError.Error())
-				return
+				require.EqualError(t, err, test.expectedError.Error())
+			} else {
+				require.NoError(t, err)
```

Comment:
I feel like require.EqualError should be sufficient here without if/else as long as you don't call `.Error()`
Comparing error type to error type should be fine IMO.

Reply:
ok.